ansible/examples/builder.yml

```diff
@@ -1016,7 +1074,7 @@
 
         - name: Install the systemd unit files for jenkins
           ansible.builtin.template:
-            src: "templates/systemd/jenkins.{{ item }}.j2"
+            src: "{{ playbook_dir }}/../templates/systemd/jenkins.{{ item }}.j2"
```

Comment:
Is this a magic variable?

Reply:
We should do the same addition to the files/ references, and remove the comment at the top starting with NOTE: that I just added

Reply:
Done, did this change on two more tasks in the playbook